Sources/Kitura-CredentialsTwitter/CredentialsTwitter.swift

```diff
@@ -0,0 +1,358 @@
+/**
+ * Copyright Jacob Van Order 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Kitura
+import KituraNet
+import LoggerAPI
+import Credentials
+
+import SwiftyJSON
+
+import Foundation
+
+
+/// CredentialsTwitter is a plugin for the Credentials framework that authenticate using Twitter. This plugin uses
+/// Twitter's [“Sign in with Twitter”](https://dev.twitter.com/web/sign-in/implementing) use case. Roughly, there are
```

Comment:
I think it's for Jazzy documentation but also creates a tappable link on the tool tip.

![screen shot 2017-01-25 at 4 29 06 pm](https://cloud.githubusercontent.com/assets/109652/22311943/74ebed20-e31b-11e6-9ac5-5935f06440fe.png)